viewer/core/src/utilities/Spinner.ts

```diff
@@ -38,12 +46,43 @@ export class Spinner {
     this.el.title = Spinner.titles.idle;
 
     this.el.className = Spinner.classnames.base;
+    this.el.classList.add(Spinner.classnames.topLeft);
     this.el.innerHTML = svg;
 
     parent.style.position = 'relative';
     parent.appendChild(this.el);
   }
 
+  set placement(placement: Corner) {
+    this.el.classList.remove(
```

Comment:
`this.el.classList.remove()` for any non existing class works fine but this might fire an error on IE 10. If we are not supporting IE 10, than this is not an issue :)